tools/cli/pkg/command/reconciliations.go

```diff
@@ -0,0 +1,43 @@
+package command
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	ErrMothershipResponse = errors.New("reconciler error response")
+)
+
+func isErrResponse(statusCode int) bool {
+	return statusCode < http.StatusOK || statusCode >= http.StatusMultipleChoices
+}
+
+func responseErr(resp *http.Response) error {
+	var msg string
+	if err := json.NewDecoder(resp.Body).Decode(&msg); err != nil {
+		msg = "unknown error"
```

Comment:
We should include the `err` in the msg - maybe rename "unknown" to "unexpected" :)